fossunited/api/hackathon.py

```
@@ -612,6 +612,79 @@ def get_issue_pr_title(url: str) -> dict:
+       intended recipient.
+    4. Checks that the request is still in ``Pending`` state so that an already
+       resolved request cannot be changed again.
+    5. Updates the status with ``ignore_permissions=True`` so that the
+       controller's ``before_save`` side-effects can run without a permission
+       error, then commits the transaction.
+    6. Returns a success dict on completion.
+
+    Args:
+        request_id (str): Name (primary key) of the
+            ``FOSS Hackathon Join Team Request`` document.
+        status (str): New status – must be ``"Accepted"`` or ``"Rejected"``.
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)